app/data.py

```diff
@@ -50,9 +50,9 @@ def delete(self, collection: str, query: Dict):
         """ Delete existing documents in collection matching given data """
         self.collection(collection).delete_many(query)
 
-    def count(self, collection: str, query: Optional[Dict] = None) -> int:
+    def count(self, collection: str, query: Optional[Dict]) -> int:
         """ Returns the number of documents in collection that matches query """
-        return self.collection(collection).count_documents(query or {})
+        return self.collection(collection).count_documents(query or {5})
```

Comment:
This change isn't going to do what you want, let's get that reverted to the original code.